gateway/engine/beans/pom.xml

```diff
@@ -12,6 +12,10 @@
 
   <dependencies>
     <dependency>
+      <groupId>net.openhft</groupId>
+      <artifactId>zero-allocation-hashing</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
I didn't realize there was a new dependency - did you check with Nick to see if this was usable?